test/core.jl

```diff
@@ -23,9 +23,8 @@
     end
 
     @testset "Hybrid NB" begin
-
         # a test to check that HybridNB successfully replaces KernelNB
-        m1 = HybridNB(y, 2)
+        m1 = HybridNB(y, ["c1", "c2"])
```

Comment:
I thought about using symbols exclusively as it's done, say, in DataFrames.jl, and converting them to strings when writing to HDF5. But again, it's you and not me who primarily uses this feature, so I make proposals, but eventually rely on your opinion.